### DISCUSSION TITLE
GRUB bootloader issue started appearing on linux machine (with no code change)

### DISCUSSION THREAD

**Discussion by gunghoiguana** [codemagic-ci-cd/codemagic-docs#1365] [Q&A] [answered]
Sometime over the weekend, all of our Android builds (performed on linux machines) started timing out with this error. It's occurring during our install step (at the end of the command `sudo apt-get install dart`).

```
The GRUB boot loader was previously installed to a disk that is no longer 
present, or whose unique identifier has changed for some reason. It is important
to make sure that the installed GRUB core image stays in sync with GRUB modules 
and grub.cfg. Please check again to make sure that GRUB is written to the 
appropriate boot devices.

  1. /dev/sda (107374 MB; ) on 107374 MB PersistentDisk
  2. /dev/sda15 (111 MB; /boot/efi) on 107374 MB PersistentDisk
  3. none of the above

(Enter the items you want to select, separated by spaces.)

GRUB install devices: 
```

There's been no code change on our part (if we run a build today, off of code that worked on Friday, the error appears).

Here's the build id of the last successful build: 63d47e58486a472b668efc51
And one that had the GRUB boot loader issue: 63d8177d543447494f7035ec

**Comment by K-Codemagic** [comment, marked as answer]
Hi, we are looking into it and will get back to you

**Reply by gunghoiguana** [reply]
This issue has been fixed for us (and it was pointed out that we don't need to be re-installing dart ourselves). Thank you!
